bokeh/models/tools.py

```diff
@@ -174,6 +174,23 @@ class PanTool(Drag):
     """)
 
 
+class WheelPanTool(Scroll):
+    """ *toolbar icon*: |wheel_pan_icon|
+
+    The wheel pan tool allows the user to pan the plot along the configured
+    dimension using the scroll wheel.
+
+    .. |wheel_pan_icon| image:: /_images/icons/WheelPan.png
+        :height: 18pt
+
+    """
+
+    dimension = Enum(Dimension, default="width", help="""
+    Which dimension the wheel pan tool is constrained to act in. By
+    default the wheel pan tool will pan the plot along the x-axis.
+    """)
```

Comment:
Tho FYI all `Enum` automatically self-document at least the options:

<img width="686" alt="screen shot 2016-08-31 at 3 36 18 pm" src="https://cloud.githubusercontent.com/assets/1078448/18145417/c11ffa6e-6f90-11e6-9e74-50ef5ca51f1f.png">